echopype/convert/parse_base.py

```diff
@@ -132,19 +131,7 @@ def parse_raw(self):
                         # and in the form of floats separated by semicolons
                         v["pulse_duration"] = [float(x) for x in v["pulse_length"].split(";")]
 
-            # If exporting to XML file (EK80/EA640 only), print a message
-            if "print_export_msg" in self.data_type:
-                if "ENV" in self.data_type:
-                    xml_type = "environment"
-                elif "CONFIG" in self.data_type:
-                    xml_type = "configuration"
-                logger.info(f"exporting {xml_type} XML file")
-                # Don't parse anything else if only the config xml is required.
-                if "CONFIG" in self.data_type:
-                    return
-            # If not exporting to XML, print the usual converting message
-            else:
-                self._print_status()
+            self._print_status()
```

Comment:
Tiny addition of a comment:
```suggestion
            # print the usual converting message
            self._print_status()
```